assets/src/components/PostTypeSelector/PostTypeSelector.js

```diff
@@ -1,44 +1,44 @@
-import { Component } from '@wordpress/element';
-import { compose } from '@wordpress/compose';
-import { withSelect } from "@wordpress/data";
+import {Component} from '@wordpress/element';
+import {compose} from '@wordpress/compose';
+import {withSelect} from '@wordpress/data';
 import {
   FormTokenField,
 } from '@wordpress/components';
 
 class PostTypeSelector extends Component {
-  constructor( props ) {
-    super( props );
+  constructor(props) {
+    super(props);
 
-    this.handleChange = this.handleChange.bind( this );
+    this.handleChange = this.handleChange.bind(this);
   }
 
-  handleChange( value ) {
-    const postTypeIds = value.map( postTypeName => {
-      return this.props.postTypes.find( postType => postType.name === postTypeName ).id;
+  handleChange(value) {
+    const postTypeIds = value.map((postTypeName) => {
+      return this.props.postTypes.find((postType) => postType.name === postTypeName).id;
     });
-    this.props.onChange( postTypeIds );
+    this.props.onChange(postTypeIds);
   }
 
   render() {
-    const { postTypes, onChange, label, placeholder, value, ...ownProps } = this.props;
+    const {postTypes, label, placeholder, value, ...ownProps} = this.props;
```

Comment:
Same issue here as for `TagSelector`, we need to exclude `onChange` from the `ownProps` object to avoid overriding the function that we pass down